app/models/creates_application_from_draft.rb

```diff
@@ -1,4 +1,5 @@
 class CreatesApplicationFromDraft
+  PROJECT_FIELDS = ApplicationDraft::PROJECT_FIELDS.map(&:to_s)
```

Comment:
Unfortunately not - it does not have *"indifferent access"* and since it's serialized down into an `hstore` column, it does only work with String.